app/controllers/user_event_preferences_controller.rb

```diff
@@ -0,0 +1,61 @@
+class UserEventPreferencesController < ApplicationController
+  before_action :set_user_event_preference, only: %i[ show edit update destroy ]
+
+  def index
+    @user_event_preferences = UserEventPreference.all
+  end
+
+  def show
+  end
```

Comment:
are you sure you need the show and edit methods? I guess you are not going to change a single preference.. or will you?